x-pack/plugins/maps/ui_tests/tests/full_screen_mode.spec.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, tags } from '@kbn/scout';
+import { test } from '../fixtures';
+
+test.describe('maps full screen mode', { tag: tags.ESS_ONLY } , () => {
+  test.beforeEach(async ({ browserAuth, pageObjects }) => {
+    await browserAuth.loginAsViewer();
+    await pageObjects.gis.goto();
+  });
+  test('full screen button should exist', async ({ pageObjects}) => {
+    await pageObjects.gis.fullScreenModeMenuItemExists();
+  });
+  test('full screen mode hides the kbn app wrapper', async ({ page, pageObjects }) => {
+    expect(await page.testSubj.locator('kbnAppWrapper visibleChrome').waitFor({state: 'visible'}));
```

Comment:
why do we need to check it multiple times? Is it something specific to maps rendering or common loading spinner?

Reply:
No idea, as I mentioned previously, this pr is not ready for review.